framework/src/Volo.Abp.BlazoriseUI/AbpCrudPageBase.cs

```diff
@@ -356,20 +357,28 @@ protected virtual Task CloseEditModalAsync()
             return Task.CompletedTask;
         }
 
+        protected virtual async Task RunProcess(Func<Task> process)
```

Comment:
Why this method still exists? Shouldn't we remove this?

Reply:
I was under impression that it should stay last time we spoke. I will remove it.

Reply:
Removed